plugin/src/sbt-test/sbt-bazel/external/BUILD.expect

```diff
@@ -7,10 +8,8 @@ load(
 scala_library(
   name = 'hello',
   deps = [
-    '//external:jar/io/circe/circe_core_2_12_0_9_3',
-  ],
-  runtime_deps = [
     '//external:jar/com/chuusai/shapeless_2_12_2_3_3',
+    '//external:jar/io/circe/circe_core_2_12_0_9_3',
```

Comment:
I'm worried about the version showing up here `0_9_3`. With this, you will have to jiggle the deps to have exactly the same version as transitive users, and more over, you won't be able to mix it with bazel-deps users.

What is the thinking on this?

It seems like it will be easy to violate the 1 version principle here and have very confusing classpath issues to debug.

Reply:
Good point. I definitely want this to work with bazel-deps. I think if I drop the version, it will generate names identical to bazel-deps. I'll change this.